ntbs-service/Models/Validations/ValidationHelper.cs

```diff
@@ -80,6 +80,8 @@ public static string ValidYearLaterThanBirthYear(int birthYear)
         public const string DenotificationDateLatestToday = "Date of denotification cannot be later than today";
         public const string DenotificationReasonRequired = "Please supply a reason for denotification";
         public const string DenotificationReasonOtherRequired = "Please supply additional details for the denotification reason";
+        public const string NotificationDateRequired = "Please supply notification date";
+        public const string ShouldBeLaterThanDob = "Notification date must be later than date of birth";

```

Comment:
As this is specific to Notification, I think the name should reflect this, "NotificationDateShouldBeLaterThanDob"